web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -489,6 +490,7 @@ goog.require('gn_alert');
       $scope.fluidHeaderLayout = gnGlobalSettings.gnCfg.mods.header.fluidHeaderLayout;
       $scope.showGNName = gnGlobalSettings.gnCfg.mods.header.showGNName;
       $scope.isHeaderFixed = gnGlobalSettings.gnCfg.mods.header.isHeaderFixed;
+      $scope.isMenubarAccessible = gnGlobalSettings.gnCfg.mods.header.isMenubarAccessible;
```

Comment:
Do we require this setting? Is there any reason to don't have the menubar accessible?

Reply:
It changes the use of the menu a little bit, the setting is for users who don't like the change